db/models.py

```diff
@@ -19,8 +21,11 @@ def __str__(self) -> str:
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    actors = models.ManyToManyField(to=Actor)
-    genres = models.ManyToManyField(to=Genre)
+    actors = models.ManyToManyField(to=Actor, related_name="movie")
+    genres = models.ManyToManyField(to=Genre, related_name="movie")
```

Comment:
according to checklist from previous task its better to use plural name for related_name